code-review/README.md

```diff
@@ -22,10 +22,15 @@ Everyone
   If they are, use them with aplomb.
 * Talk in person if there are too many "I didn't understand" or "Alternative
   solution:" comments. Post a follow-up comment summarizing offline discussion.
+* Try to make small atomic PRs to allow simpler reviews. If a "feature" requires 
+  multiple PRs at once document the dependencies and in what order they should
+  be reviewed.
+* Close stale PRs.
 
 Having Your Code Reviewed
 -------------------------
 
+* Include set up and testing instructions.
```

Comment:
Suggestion: Include a link to the relevant PR in the case of a cross-repo dependent set of PRs.

Reply:
👍